### ISSUE TITLE
Error while running maker.py and taker.py files

### ISSUE THREAD

**Issue by Prajit65** [etherdelta/bots#16]
I get the following error while running the maker.py file
Traceback (most recent call last):
  File "/home/prajit/web3.py/web3/contract.py", line 748, in call_contract_function
    output_data = decode_abi(output_types, return_data)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/abi.py", line 109, in decode_abi
    return decoder(stream)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/decoding.py", line 102, in __call__
    return self.decode(stream)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_utils/functional.py", line 22, in inner
    return callback(fn(*args, **kwargs))
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/decoding.py", line 140, in decode
    yield decoder(stream)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/decoding.py", line 102, in __call__
    return self.decode(stream)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/decoding.py", line 165, in decode
    raw_data = cls.read_data_from_stream(stream)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/decoding.py", line 247, in read_data_from_stream
    len(data),
eth_abi.exceptions.InsufficientDataBytes: Tried to read 32 bytes.  Only got 0 bytes

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "maker.py", line 28, in <module>
    es.printBalances(token, userAccount)
  File "/home/prajit/etherdeltaclientservice.py", line 184, in printBalances
    print("Wallet token balance: %.18f tokens" % self.getBalance(token, userAccount))
  File "/home/prajit/etherdeltaclientservice.py", line 77, in getBalance
    balance = self.contractToken.call().balanceOf(user)
  File "/home/prajit/web3.py/web3/contract.py", line 770, in call_contract_function
    raise BadFunctionCallOutput(msg) from e
web3.exceptions.BadFunctionCallOutput: Could not decode contract function call balanceOf return data 0x for output_types ['uint256']

This is the error i get while running taker.py file

Traceback (most recent call last):
  File "/home/prajit/web3.py/web3/contract.py", line 748, in call_contract_function
    output_data = decode_abi(output_types, return_data)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/abi.py", line 109, in decode_abi
    return decoder(stream)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/decoding.py", line 102, in __call__
    return self.decode(stream)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_utils/functional.py", line 22, in inner
    return callback(fn(*args, **kwargs))
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/decoding.py", line 140, in decode
    yield decoder(stream)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/decoding.py", line 102, in __call__
    return self.decode(stream)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/decoding.py", line 165, in decode
    raw_data = cls.read_data_from_stream(stream)
  File "/home/prajit/venv/lib/python3.5/site-packages/eth_abi/decoding.py", line 247, in read_data_from_stream
    len(data),
eth_abi.exceptions.InsufficientDataBytes: Tried to read 32 bytes.  Only got 0 bytes

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "taker.py", line 24, in <module>
    es.printBalances(token, userAccount)
  File "/home/prajit/etherdeltaclientservice.py", line 184, in printBalances
    print("Wallet token balance: %.18f tokens" % self.getBalance(token, userAccount))
  File "/home/prajit/etherdeltaclientservice.py", line 77, in getBalance
    balance = self.contractToken.call().balanceOf(user)
  File "/home/prajit/web3.py/web3/contract.py", line 770, in call_contract_function
    raise BadFunctionCallOutput(msg) from e
web3.exceptions.BadFunctionCallOutput: Could not decode contract function call balanceOf return data 0x for output_types ['uint256']

Basically they are the same errors. Can someone help me figure out whats the problem?
